src/vmware/azext_vmware/_params.py

```diff
@@ -25,11 +25,13 @@ def load_arguments(self, _):
         c.argument('circuit_secondary_subnet', help='A /30 subnet for the secondary circuit in the Express Route to configure routing between your network and Microsoft\'s Enterprise edge (MSEEs) routers.')
         c.argument('cluster_size', help='Number of hosts for the default management cluster. Minimum of 3 and maximum of 16.')
         c.argument('network_block', help='A subnet at least of size /22. Make sure the CIDR format is conformed to (A.B.C.D/X) where A,B,C,D are between 0 and 255, and X is between 0 and 22.')
+        c.argument('identity_type', help='The type of identity used for the private cloud. The type "SystemAssigned" refers to an implicitly created identity. The type "None" will remove any identities from the Private Cloud.')

```

Comment:
Please refer to the doc [managed_identity_command_guideline](https://github.com/Azure/azure-cli/blob/dev/doc/managed_identity_command_guideline.md) to design managed identity related parameters

Reply:
Updated based on guidelines